src/components/CourseSearchResults.js

```diff
@@ -31,4 +35,10 @@ const SearchFailedMessage = () => (
     </div>
 );
 
+const NoSearchResultsMessage = () => (
+    <div className="c-alert c-alert--info">
```

Comment:
I think we could do this with a pattern Library `<Alert>` component with a className of info
```suggestion
    <Alert className="info">Sorry, your search did not return any results.</Alert>
```

Reply:
I didn't clock this, yes this feels like an improvement (although an `InfoAlert` would be even better)! As we have another alert already being used elsewhere, I'll look at refactoring them both at the same time in a separate commit